check.go

```diff
@@ -173,21 +173,21 @@ type CheckAlertSourceUsageUpdateInput struct {
 type CheckCustomEventCreateInput struct {
 	CheckCreateInput
 
-	Integration      graphql.ID `json:"integrationId"`
-	ServiceSelector  string     `json:"serviceSelector"`
-	SuccessCondition string     `json:"successCondition"`
-	Message          string     `json:"resultMessage,omitempty"`
-	PassPending      *bool      `json:"passPending,omitempty"`
+	Integration      ID     `json:"integrationId"`
+	ServiceSelector  string `json:"serviceSelector"`
+	SuccessCondition string `json:"successCondition"`
+	Message          string `json:"resultMessage,omitempty"`
+	PassPending      *bool  `json:"passPending,omitempty"`
 }
 
 type CheckCustomEventUpdateInput struct {
 	CheckUpdateInput
 
-	ServiceSelector  string      `json:"serviceSelector,omitempty"`
-	SuccessCondition string      `json:"successCondition,omitempty"`
-	Message          string      `json:"resultMessage,omitempty"`
-	PassPending      *bool       `json:"passPending,omitempty"`
-	Integration      *graphql.ID `json:"integrationId,omitempty"`
+	ServiceSelector  string `json:"serviceSelector,omitempty"`
+	SuccessCondition string `json:"successCondition,omitempty"`
+	Message          string `json:"resultMessage,omitempty"`
+	PassPending      *bool  `json:"passPending,omitempty"`
+	Integration      *ID    `json:"integrationId"`
```

Comment:
i don't feel like this is correct - we should inspect this one and make sure the tests are good.

Reply:
Ok, I'll look at the tests for this one.